package.json

```diff
@@ -0,0 +1,252 @@
+{
+    "name": "mindaro",
+    "displayName": "Bridge to Kubernetes",
+    "description": "Rapid Kubernetes development for teams",
+    "version": "1.0.1",
+    "icon": "images/bridge128.png",
+    "preview": false,
+    "publisher": "mindaro",
+    "qna": "marketplace",
+    "license": "MIT",
```

Comment:
Should we include a MIT license in the repo? I'm not sure if this is required in this case.

Reply:
Thank you for this mention by default the whole repo is, please let me know if there is anything specific we can change. 🙏☕️

<img width="289" alt="Screen Shot 2022-08-24 at 9 34 48 PM" src="https://user-images.githubusercontent.com/6233295/186385075-1a74ed05-3e26-43c1-955e-df08a641e993.png">

Reply:
Ah OK didn't see that. Thanks!